readme.md

```diff
@@ -286,7 +286,7 @@ const value = useFormFieldValue(field)
 Subscribes to the values of multiple fields (or forms).
 
 ```jsx
-const values = useFormFieldsValues(fields)
+const [...values] = useFormFieldsValues([...fields])
```

Comment:
Wat is hier precies de gedachte achter?

Reply:
Om aan te geven dat de interface met arrays werkt.

Reply:
Is hier geen typing voor aanwezig? Ik vind het zelf wel handig, maar ben bang dat we in de PRs dan letterlijk het `[...[]]` patroon terug gaan zien.

Reply:
Er zijn typings voor de schema functies maar verder geen. Maar ik haal dit wel weg.